web/client/api/catalog/__tests__/CSW-test.js

```diff
@@ -302,7 +302,34 @@ describe('Test correctness of the CSW catalog APIs', () => {
         }, {});
         expect(records.length).toBe(1);
     });
-
+    it('csw with DC URI 3D Tiles', ()=>{
+        const records = [{
+            boundingBox: {
+                "extent": [
+                    43.718, 11.348, 43.84, 11.145
+                ],
+                "crs": "EPSG:4326"
+            },
+            dc: {
+                URI: {
+                    TYPE_NAME: "DC_1_1.URI",
+                    description: "access point",
+                    protocol: "https://registry.geodati.gov.it/metadata-codelist/ProtocolValue/www-download",
+                    value: "https://3d-layers.s3.eu-central-1.amazonaws.com/3dtiles/centro_storico_di_firenze_-_brass_city_model/tileset.json"
+                }, format: '3D Tiles', identifier: "test:20230829_test_3dtile_01", title: "3D Tiles layer for test"
```

Comment:
format can be the constant 3D_TILES
```suggestion
                }, format: '3D Tiles', identifier: "test:20230829_test_3dtile_01", title: "3D Tiles layer for test"
```